src/dpr/components/show-more/clientClass.mjs

```diff
@@ -0,0 +1,42 @@
+import { DprClientClass } from '../../DprClientClass.mjs'
+
+export default class ShowMore extends DprClientClass {
+  static getModuleName() {
+    return 'show-more'
+  }
+
+  initialise() {
+    this.showMoreComponents = document.querySelectorAll('.dpr-show-more')
```

Comment:
An instance of this class is created for each element with the `data-dpr-module` set, and the specific element is accessible via the `getElement()` function.

As it currently is, if there are multiple `ShowMore` elements, then each one will try to initialise all of them.

Initialisation: https://github.com/ministryofjustice/hmpps-digital-prison-reporting-frontend/pull/191/files#diff-9a21e0a5e02fe9b2482d2e549aecaa56bb97faa53728f7f571c2ee15b5ba7148R55

Reply:
Thanks! getElement was what I needed. I tried to fix this with a initialisation check but this is much better. Silly me